perf_dashboard/static/js/cpu_memory.js

```diff
@@ -330,7 +330,7 @@ new Chart(document.getElementById("mem-qps-master"), {
                 hidden: true,
                 fill: false
             }, {
-                label: "none-both",
+                label: "none-mtls-both",
```

Comment:
what is none? I thought none meant plain text?

Reply:
none means without mixer, plaintext is plaintext